package.json

```diff
@@ -73,6 +73,7 @@
     "@babel/runtime": "^7.0.0",
     "@videojs/http-streaming": "1.4.2",
     "global": "4.3.2",
+    "keyboard-key": "^1.0.2",
```

Comment:
No specific experience with this module. Yeah, https://www.npmjs.com/package/keycode looks like a viable alternative (I'd forgotten that you'd mentioned it in the previous issue), I just went with this one because it made sense.

Do you want me to try that other one?

Reply:
Yeah, maybe. Would be good to see if keycode provides everything we need and how the filesize difference affects Video.js bundles.